lib/Doctrine/ORM/Mapping/Driver/DatabaseDriver.php

```diff
@@ -79,12 +80,18 @@ class DatabaseDriver implements MappingDriver
      */
     private $namespace;
 
+    /**
+     * @var \Doctrine\Inflector\Inflector
+     */
+    protected $inflector;
```

Comment:
Since there is an accessor, it can (and should?) be private

```suggestion
    private $inflector;
```

Reply:
No, in this case its used from protected methods, maybe someone overwrites them and needs access.